index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main id="main">
+        <h1 id="title">Raila Odinga </h1>
+        <p>Studied at Leipzig University</p>
+        <figure id="img-div">
+            <img id="image" src="images/Odinga.jpeg" alt="#" />
+            <figcaption id="img-caption">
+                Raila Odinga was born at the Anglican Church Missionary Society Hospital,
+                in Maseno, to Mary Juma Odinga and Jaramogi Oginga Odinga. His father served as the first Vice President
+                of Kenya under President Jomo Kenyatta. He is a member of the Luo ethnic group.
+            </figcaption>
+        </figure>
+        <section id="tribute-info">
+            <h3 id="headline">The following is Raila Odinga life:</h3>
+            <ul>
+                <li><strong>1945</strong> - Kisumu District, Nyanza Province on 7 January 1945</li>
```

Comment:
- [ ] Good that you used inline comment for your review @mj-e-boc :+1:  I suggest that your review should includes an explanation, a "why", for each requested change. :blush: I think you should checks the code to look for errors and bugs and commently clearly the suggested change. Examples of issues to check when performing the review: :relaxed: 
  
  * Lack of [semantic HTML tags use](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
  * Not applying [HTML & CSS best practices correclty](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
  * Bugs missed the first time the project was built.

- [ ] I would be better if you phrase the comments politely, as suggestions instead of orders. For example: :point_down: 
  
  * This is a polite suggestion: *"I think it would be better to use semantic tags"*
  * This reads as an order: *"Use semantic tags"*